app/controllers/games_controller.rb

```diff
@@ -31,33 +30,16 @@ def play_game
   def show
     resume_game = Game.where(:id => params[:id]).first
```

Comment:
You should be able to achieve the same result with `Game.find_by(id: params[:id])`.